cmd/eval.go

```diff
@@ -168,11 +168,11 @@ Examples
 
 To evaluate a simple query:
 
-    $ opa eval 'x = 1; y = 2; x < y'
+    $ opa eval 'x := 1; y := 2; x < y'
 
 To evaluate a query against JSON data:
 
-    $ opa eval --data data.json 'data.names[_] = name'
+    $ opa eval --data data.json 'name := data.names[_]'
```

Comment:
Alternatively, but a bit of a mouthful:
```
opa eval --import future.keywords.in --data data.json "some name in data.names"
```

Reply:
Yeah, I considered it, but I agree that it's a bit much for a potentially first encounter of the eval command.